Automattic/woocommerce-payments
# Replace custom tour component with TourKit from @woocommerce/components

Fixes Automattic/woocommerce-payments-server#3136

#### Changes proposed in this Pull Request









Replace the custom tour implementation with the TourKit component from `@woocommerce/components`. The custom component was created before knowing about the existence of this component, so we're removing the custom implementation in favor of it.

#### Testing instructions





- Checkout to this branch `update/3136-welcome-tour`
- Run `npm start`
- Enable the `wcpay_fraud_protection_settings_active` flag on your client
  - If you don't have it set yet, run: `npm run wp option add wcpay_fraud_protection_settings_active 1`
  - Otherwise, update it by running: `npm run wp option update wcpay_fraud_protection_settings_active 1`
- Make sure the `wcpay_fraud_protection_welcome_tour_dismissed` is disabled
  - If you don't have it set yet, run: `npm run wp option add wcpay_fraud_protection_welcome_tour_dismissed 0`
  - Otherwise, update it by running: `npm run wp option update wcpay_fraud_protection_welcome_tour_dismissed 0`
- Go to **Payments > Settings**
- The welcome tour should pop up
- Go through the tour steps and finish it – or click on the close button
- Refresh the page – it shouldn't pop up again
- Check the welcome tour flag by running `npm run wp option get wcpay_fraud_protection_welcome_tour_dismissed`
  - It should return `1`
- If you want to test it again, run the following command
  - `npm run wp option update wcpay_fraud_protection_welcome_tour_dismissed 0`
- Refresh the page – the tour should pop up again

-------------------


**Post merge**
